packages/blockchain-common/test/fabricModel/FabricNode.test.ts

```diff
@@ -103,15 +103,6 @@ describe('FabricNode', () => {
             }
         });
 
-        it('should throw an error if doesn\'t have a name', () => {
```

Comment:
Why has this test been removed?

Reply:
Because it's duplicated, probably from a merge conflict. I didn't think a new PR was justified for just that change.